src/helpers/gpt.ts

```diff
@@ -0,0 +1,180 @@
+import { getBotConfig, getBotContext, getLogger } from "../bindings";
+import { Payload, StreamlinedComment, UserType } from "../types";
+import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody } from "../helpers";
+import OpenAI from "openai";
+import { CreateChatCompletionRequestMessage } from "openai/resources/chat";
+import { ErrorDiff } from "../utils/helpers";
+
+export const sysMsg = `You are the UbiquityAI, designed to provide accurate technical answers. \n
+Whenever appropriate, format your response using GitHub Flavored Markdown. Utilize tables, lists, and code blocks for clear and organized answers. \n
+Do not make up answers. If you are unsure, say so. \n
+Original Context exists only to provide you with additional information to the current question, use it to formulate answers. \n
+Infer the context of the question from the Original Context using your best judgement. \n
+All replies MUST end with "\n\n <!--- { 'UbiquityAI': 'answer' } ---> ".\n
+`;
+
+export const gptContextTemplate = `
+You are the UbiquityAI, designed to review and analyze pull requests.
```

Comment:
I think the command is not registered. I couldn't find it earlier.

Reply:
@Keyrxng rfc

Reply:
My mistake fellas, forgot to replace it following shuffling the order of the contexts. In gpt.ts inside /review, the template is supplied first before anything else [#L202](https://github.com/Keyrxng/ubiquibot/blob/f1400cc00b500ac260d6c53c6fb60c6c858e52d7/src/helpers/gpt.ts#L202). It's not used in /review as we are only passing issue/pull body as context now but it should be passed in to chatHistory for ask.

Without it GPT responds in a story telling way rather than a json format but achieves the same thing, but JSON does provide more data for the dependant call.

Let me know what happens next, whether I'm to fix it or what bit wet behind the ears with this

Reply:
You can file a new issue and start working on it to get paid. If its too small of a task it might be more worth to just open the pull request for it without filing an issue. Either way is fine. 

I should be able to price it before you get it done.

Reply:
No no new issue or anything needed, it's just adding one convo object into chatHistory but I appreciate that. I'll have time after 6pm tonight to catch it, don't trust doing it on my phone lmao too many fat thumbs waiting to happen